parsing/utils.py

```diff
@@ -0,0 +1,60 @@
+from enum import Enum
+
+
+class TelegramChannel:
```

Comment:
Everybody loves `utils.py` :D
But let's give this file some meaningful name, keeping in mind that in the future other parsers will be added here besides Telegram. Maybe let's group them into folder, idk.